src/containers/Home.js

```diff
@@ -210,43 +210,60 @@ class Homepage extends React.Component {
             { property: 'og:url', content: SITE_META.URL }
           ]}
         />
-        <SideBar>
+        <SideBar
+          moduleLabelObj={moduleLabelObj}
+        >
           <FirstModuleWrapper
             moduleId={moduleIdObj.editorPick}
```

Comment:
Put anchor name here. Delete `moduleLabelObj`.
`moduleIdObj` could be defined here instead of in `twreporter-react-index-page-component`